ui.go

```diff
@@ -317,8 +340,11 @@ func (c *chat) renderUsers(dggusers []dggchat.User) {
 
 		var users string
 		for _, u := range dggusers {
```

Comment:
Probably rename all occurrences to just `users` if applicable. If this will make it harder to cherry-pick further commits, disregard for now.